### PR TITLE
add cancel-all CLI endpoint

### PR DESCRIPTION
**Pull request by mmou**

### CODE REVIEW

File: go/client/cmd_wallet_cancel_all.go [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+			}
+		}(txID)
+	}
+
+	wg.Wait()
+	dui.Printf("Finished canceling. successCount=%d, skipCount=%d, failCount=%d, writeErrorCount=%d\n\n", successCount, skipCount, failCount, writeErrorCount)
+	return nil
+}
+
+func (c *cmdWalletCancelAll) GetUsage() libkb.Usage {
```
Comment (mmou) [inline go/client/cmd_wallet_cancel_all.go]: is this how to unlist the cmd?

Comment (mlsteele) [reply go/client/cmd_wallet_cancel_all.go]: Use `Unlisted: true` like [this](https://github.com/keybase/client/blob/master/go/client/cmd_account_reset_cancel.go#L21).

To be honest I don't know what `GetUsage` is for, but probably copy what [cmdWalletCancel does](https://github.com/keybase/client/blob/master/go/client/cmd_wallet_cancel.go#L63:L63)

Comment (mmou) [reply go/client/cmd_wallet_cancel_all.go]: done

File: go/client/cmd_wallet_cancel_all.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+			defer func() {
+				sem.Release(1)
+				wg.Done()
+			}()
+			res, err := cli.ClaimCLILocal(context.TODO(), stellar1.ClaimCLILocalArg{TxID: txID})
+			if err != nil {
+				// you may want to add more cases for when an erroring txID should be
+				// added to out.txt
+				if strings.Contains(err.Error(), "Payment already claimed by ") {
+					dui.Printf("SKIP: %+v, %+v\n", txID, err.Error())
```
Comment (patrickxb) [inline go/client/cmd_wallet_cancel_all.go]: could you switch these `%+v` to `%s` when you know it is a string (here and below)?  also, you don't need to do `err.Error()` here, just `err` will be fine.

File: go/client/cmd_wallet_cancel_all.go [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+		dui.Printf("Note: no outfile specified, so not keeping track of complete txIDs.\n")
+	}
+
+	maxWorkers := 100 // OR runtime.GOMAXPROCS(0)
+	sem := semaphore.NewWeighted(int64(maxWorkers))
+	dui.Printf("Canceling %d txIDs with %d maxWorkers; %d txIDs already complete...\n", len(in), maxWorkers, len(out))
+
+	var wg sync.WaitGroup
+	var successCount, skipCount, failCount, writeErrorCount int
+	for _, txID := range in {
```
Comment (patrickxb) [inline go/client/cmd_wallet_cancel_all.go]: could you trap SIGTERM, SIGINT to cleanly exit this loop (and thus gracefully shut down this command)?

Comment (mmou) [reply go/client/cmd_wallet_cancel_all.go]: done

**Pull request merged**